ChatRPG/ChatRPG/Data/Models/CharacterEnvironment.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ChatRPG.Data.Models;
+
+public class CharacterEnvironment
+{
+    private CharacterEnvironment() {}
+
+    public CharacterEnvironment(Campaign campaign, Character character, Environment environment, int version)
+    {
+        Campaign = campaign;
+        Character = character;
+        Environment = environment;
+        Version = version;
```

Comment:
Kan vi ikke kalde det noget andet end version? Jeg synes det lyder lidt mærkeligt, for det virker til at version betyder "det nyeste sted", men jeg føler at version burde betyde "den nyeste beskrivelse/opfattelse af Castle Ravenloft" f.eks.

Det virker mere til at det er en konologisk rækkefølge af hvordan vi har besøgt environments.
Giver min kommentar mening?

Reply:
Ift. ERD kan jeg se at character og version er PK (nok også campaign, men det er ikke på atm) og environment er FK. Dermed er jeg enig i at det ligner en kronologisk logging af hvor character har været. Er der behov for at vi har historik over hvor de har været, frem for bare hvor de er (nuværende)?

Reply:
@mirakst @KarmaKamikaze Jeg overvejede også selv om man bare skal gemme den nuværende lokation.
Idéen bag historik er at man kan generere et "kort" over spillerens rejse. Men det ser jeg også som en "could have" feature, nu hvor jeg tænker over det.
Hvis vi beholder historikken, er jeg enig i at `Version` skal omdøbes. Ellers skal tabellen bare fjernes.

Reply:
@Sarmisuper @selectionator Nogen der er **imod** at vi kun gemmer den nuværende lokation for en character?
Vi kan altid udvide dette senere (men jeg forestiller mig ikke at vi får tid til det).

Reply:
Nope

Reply:
_it is done_